### PR TITLE
Examples: mobile support for unreal_bloom_selective example

### PR DESCRIPTION
**Pull request by ajflores1604**
Changed event listener and corresponding callback name from 'click' to 'pointerdown' so demo works on mobile. 
https://raw.githack.com/ajflores1604/three.js/unreal_bloom_selective_mobile/examples/webgl_postprocessing_unreal_bloom_selective.html

### CODE REVIEW

File: examples/webgl_postprocessing_unreal_bloom_selective.html [hunk 1]
```diff
@@ -125,7 +125,7 @@
 
 			var mouse = new THREE.Vector2();
 
-			window.addEventListener( 'click', onDocumentMouseClick, false );
+			window.addEventListener( 'pointerdown', onDocumentPointerDown, false );
```
Comment (Mugen87) [inline examples/webgl_postprocessing_unreal_bloom_selective.html:128]: Nit: How about renaming `onDocumentPointerDown()` do just `onPointerDown()`? `document` is not relevant here.

Comment (ajflores1604) [reply examples/webgl_postprocessing_unreal_bloom_selective.html:128]: yeah that makes more sense. I was trying to change as little as possible initially

**Review by Mugen87 — APPROVED** [review]

**Comment by ajflores1604** [comment]
Sorry, i thought i had to close to clean up. still learning haha

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**